src/format/bincode.rs

```diff
@@ -0,0 +1,34 @@
+//! Contains an implementation of CBOR serialization format.
```

Comment:
Is this related to the msg queue changes? I don't see the connection

Reply:
It should be my mistake...
My branch is a bit ahead and i dont know how to make PR with isolated commit, could you help?

Reply:
Sure, will fix!